src/app/utils/ServiceUtils.ts

```diff
@@ -69,19 +108,85 @@ const ServiceUtils = {
 		};
 		return (new Date(dateIso)).toLocaleString('en-US', options);
 	},
-	
-	validateLevel2AboveMembers(guildMember: GuildMember): void {
-		const isLevel2 = ServiceUtils.isLevel2(guildMember);
-		const isLevel3 = ServiceUtils.isLevel3(guildMember);
-		const isLevel4 = ServiceUtils.isLevel4(guildMember);
-		const isGenesisSquad = ServiceUtils.isGenesisSquad(guildMember);
-		const isAdmin = ServiceUtils.isAdmin(guildMember);
 
-		if (!(isLevel2 || isLevel3 || isLevel4 || isAdmin || isGenesisSquad)) {
-			throw new ValidationError('Must be `level 2` or above member.');
+	/**
+	 * Bans a guild member if they have a nickname or username similar to that of a high ranking member 
+	 * of the Discord. 
+	 * 
+	 * @param member guild member object
+	 * @returns boolean indicating if user was banned
+	 */
+	async runUsernameSpamFilter(member: GuildMember): Promise<boolean> {
+		// Only enabled for BanklessDAO server
+		if (member.guild.id !== process.env.DISCORD_SERVER_ID) {
```

Comment:
Can we extend this for the discord bot garage too?

Reply:
It should work as long as the DISCORD_SERVER_ID is set to the garage in the QA env variables.